### PR TITLE
Setup infrastructure

### PR DESCRIPTION
**Pull request by janhohenheim**
Add the following files.

- [x] .gitignore
- [x] Readme
- [x] MIT and Apache v2 licenses
- [x] Solution
- [x] Paket (and run `paket.exe install`)
- [x] .nuspec
- [x] .travis.yml

### CODE REVIEW

File: .nuspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.QueryProvider</id>
+        <version>0.1.0</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
+        <owners>messerli-informatik-ag</owners>
+        <projectUrl>http://github.com/messerli-informatik-ag/query-provider</projectUrl>
+        <license type="expression">MIT or Apache-2.0</license>
+        <description>A generic LINQ QueryProvider implementation, used by LinqToRest.</description>
+        <copyright>Copyright ©2019 Messerli Informatik Ag</copyright>
```
Comment (dngroth) [inline .nuspec]: AG

**Review by dngroth — CHANGES_REQUESTED** [review]

File: .nuspec [hunk 2]
```diff
@@ -10,5 +10,9 @@
         <description>A generic LINQ QueryProvider implementation, used by LinqToRest.</description>
         <copyright>Copyright ©2019 Messerli Informatik Ag</copyright>
         <tags>utility query linq</tags>
+        <dependencies>
+            <dependency id="Messerli.Utility" version="[0.1.0,1.0.0)" />
```
Comment (dngroth) [inline .nuspec:14]: `[ ... )`?

**Review by dngroth — CHANGES_REQUESTED** [review]

File: .nuspec (hunk 2 above)
Comment (janhohenheim) [reply .nuspec:14]: `[` = inclusive, `)` = exclusive

File: .nuspec (hunk 1 above)
Comment (janhohenheim) [reply .nuspec]: Addressed in https://github.com/messerli-informatik-ag/query-provider/pull/1/commits/9b871ce91afc0cfe943f2661a17ed5bbc8b2a530

**Review by dngroth — APPROVED** [review]

**Pull request merged**